Assets/Mirage/Runtime/ServerObjectManager.cs

```diff
@@ -440,6 +435,14 @@ internal void SpawnObject(GameObject obj, INetworkPlayer ownerPlayer)
             identity.RebuildObservers(true);
         }
 
+        void SetIdentityValues(NetworkIdentity identity)
```

Comment:
maybe this function should be inside NetworkIdentity, and pass in Server+SOM

Reply:
applied below